main.js

```diff
@@ -0,0 +1,74 @@
+// Tracks Board State //
+let gameBoardObj = {
+  1: [5, 4, 3, 2, 1],
+  2: [],
+  3: []
+}
+
+
+
+
+// Handles Opening Display of Game Board //
+let renderGame = function() {Object.values(gameBoardObj).map((game) => {
```

Comment:
wrong usage of map, performance wise, you don't want to use it as its objective is to return a new list/array. Here what you wanted to use is forEach that is more performant for what you want to do.